free/templates/free/apparatuses.html

```diff
@@ -31,4 +31,17 @@ <h2><i class="table icon"></i>{% trans 'List of Apparatuses' %}</h2>
     </div>
 
 {% endif %}
-{% endblock %}
\ No newline at end of file
+{% endblock %}
+
+{% block script %}
+<script>
+$(function() {
```

Comment:
Why in javascript? I would vote for doing it in the template. It's not dynamic anyway....

Reply:
Check out [633c1fb](https://github.com/e-lab-FREE/FREE_Web/pull/40/commits/633c1fba3d4bea4a200c8fc5286e177827a3e199)